### DISCUSSION TITLE
PhoenixD use cases

### DISCUSSION THREAD

**Discussion by apemithrandir** [ACINQ/phoenixd#28] [General]
Hello.
Is PhoenixD's main use case for enterprise to setup a standalone server based always available Phoenix Wallet?
Can you run PhoenixD and the Phoenix mobile application simultaneously, essentially making your Phoenix mobile application always online? Or like most implementations will issues arise from having the same Lightning wallet open on two different devices?

**Comment by dpad85** [comment]
The use case is to make it easy for a service/business to receive/send LN payments, while being self-custodial, and never have to manage liquidity/channels. It's a different product than Phoenix Wallet - even though it's based on the same stack, and use some of the same concepts. 

Take a look at the [phoenixd FAQ](https://phoenix.acinq.co/server/faq) for details, as well as the Auto liquidity page.

> Can you run PhoenixD and the Phoenix mobile application simultaneously ...

No, see: https://phoenix.acinq.co/server/faq#can-i-share-the-same-seed-between-several-instances-of-phoenix

**Comment by jasenwar** [comment]
I know you say you cannot reuse on a diff instance but if you completely remove the original instance and only have one instance on a lets say a diff server can you at that point reuse the seed?

**Reply by pm47** [reply]
>  if you completely remove the original instance and only have one instance on a lets say a diff server can you at that point reuse the seed?

Yes, that would be fine.

**Comment by apemithrandir** [comment]
If you could control PhoenixD from the Mobile App, then you would have an awesome experience. Transfer the phone instance to a PhoenixD instance and then switch to controller mode in the Phone App.

**Reply by dpad85** [reply]
You can take a look at this project (not developed by us): https://github.com/Hodladi/HodladiWallet, which is a web app remotely controlling a phoenixd server. It seems it can run as a PWA mobile app.

If you were thinking of plugging a [Phoenix Wallet](https://github.com/ACINQ/phoenix) to phoenixd, then that's not something we plan to do. Phoenix Wallet will remain a standalone application, that does not require to run a server on the side.

**Comment by dme1sc** [comment]
Anyway to build this multiple times on the same server serving different end users on a shared server?
